index.html

```diff
@@ -21,7 +21,7 @@
     <link rel="manifest" href="./manifest.json">
     <script src="./src/scripts/pwainit.js"></script>
 
-
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Jquery is included in the very next line, I don't think this is needed :)

Reply:
Yes, I have removed it.